internal/actor/user/booklend/app.go

```diff
@@ -0,0 +1,27 @@
+package booklend
+
+import (
+	"github.com/asynkron/protoactor-go/actor"
+
+	"github.com/ccamel/playground-protoactor.go/internal/app"
+	"github.com/ccamel/playground-protoactor.go/internal/eventsourcing"
+	"github.com/ccamel/playground-protoactor.go/internal/middleware"
+)
+
+type App struct {
+	middleware.SpawnAwareMixin
+}
+
+func (a *App) Receive(context actor.Context) {
+	if _, ok := context.Message().(*actor.Started); ok {
+		a.SpawnNamedOrDie(context, eventsourcing.ManagerProps(actor.PropsFromProducer(New)), "book_lend")
+	}
+}
+
+func init() {
+	app.RegisterMust("book_lend_app", func(_ any) actor.Producer {
+		return func() actor.Actor {
+			return &App{}
+		}
+	})
+}
```

Comment:
It’s a good idea, but integrating the application configurations is a bit more complex than that. That’s why I’ve left the value as nil for now (I probably should have added a TODO note for this). So, each application has its own configuration, which will likely be defined using a protobuf schema. The app controller will load the configuration from a YAML file and retrieve each entry based on the app’s name—or something along those lines. This will be part of a future improvement.